cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -600,8 +667,8 @@ void ImageFlip::disparityCallback(const stereo_msgs::DisparityImage::ConstPtr& d
 	if (rotation_angle==0. || rotation_angle==360. || rotation_angle==-360.)
 	{
 		disparity_image_turned = disparity_image;
-		rot_mat.at<double>(0,0) = 1.;
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.